CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Release Version Changelog
 
-## 1.5.2
-* Bump `org.apache.logging:log4j:log4j-api` `2.15.0` -> `2.17.0`
-* Bump `org.apache.logging:log4j:log4j-core` `2.15.0` -> `2.17.0`
-* Fixes CVE-2021-45105
+## 1.5.3
+* Bump `org.apache.logging:log4j:log4j-api` `2.17.0` -> `2.17.1`
+* Bump `org.apache.logging:log4j:log4j-core` `2.17.0` -> `2.17.1`
+* Fixes CVE-2021-44832
```

Comment:
```suggestion
## 1.5.3
* Bump `org.apache.logging:log4j:log4j-api` `2.17.0` -> `2.17.1`
* Bump `org.apache.logging:log4j:log4j-core` `2.17.0` -> `2.17.1`
* Fixes CVE-2021-44832

## 1.5.2
* Bump `org.apache.logging:log4j:log4j-api` `2.15.0` -> `2.17.0`
* Bump `org.apache.logging:log4j:log4j-core` `2.15.0` -> `2.17.0`
* Fixes CVE-2021-45105
```

Reply:
seems to me you deleted the previous version info

Reply:
nice catch ty 👍